### DISCUSSION TITLE
DiffPooling example code

### DISCUSSION THREAD

**Discussion by mdanb** [pyg-team/pytorch_geometric#2239] [Q&A] [answered]
I'm looking at the [example code](https://github.com/rusty1s/pytorch_geometric/blob/master/examples/proteins_diff_pool.py) provided in the Github repo for DiffPooling, and I'm a bit confused about the output of the model. The final column dimensions of `x` in the `forward` function is 6 (after `x = self.lin2(x)`, but the `PROTEINS` dataset only has two classes. Was wondering if someone could clarify what's going on.

**Comment by rusty1s** [comment, marked as answer]
This is indeed wrong, although it does not do any harm having more outputs than classes :) I will fix it.
